octavia-cli/octavia_cli/apply/resources.py

```diff
@@ -414,13 +494,15 @@ def factory(api_client: airbyte_api_client.ApiClient, workspace_id: str, configu
         NotImplementedError: Raised if the definition type found in the YAML is not a supported resource.
 
     Returns:
-        Union[Source, Destination]: The resource object created from the YAML config.
+        Union[Source, Destination, Connection]: The resource object created from the YAML config.
     """
     with open(configuration_path, "r") as f:
         local_configuration = yaml.load(f, yaml.FullLoader)
     if local_configuration["definition_type"] == "source":
```

Comment:
```suggestion
    if local_configuration["definition_type"] == DEFINITION_TYPE.source:
```
wdyt creating a enum class for definitions? I think there is other places to be used.
only small suggestion though